### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by Sayhooooooo** [List-KR/community#16] [General]
애드가드 앱 설치 후 mbp.hanabank.com에 진입이 안되요 도와주세요

I can't enter mbp.hanabank.com after installing the AdGuard app. Please help me

**Comment by piquark6046** [comment]
AdGuard DNS 서버를 AdGuard 앱에서 사용하시고 계시나요?
만약 그렇다면 임시 방편으로 Cloudflare DNS 서버를 사용해보시겠어요?

**Comment by piquark6046** [comment]
https://github.com/List-KR/List-KR/discussions/732#discussioncomment-6292291 를 참고해주세요.
